index.html

```diff
@@ -204,44 +204,47 @@ <h2 id="media-text">
         </div>
         <div class="languages">
             <h3 id="lang">
-                <p>Languages</p>
-                <img src="images\Union.svg">
+                <p role="article">Languages</p>
+                <img src="images\Union.svg" alt="">
             </h3>
             <ul class="coding-languages-images">
                 <li><img src="images\java.svg" alt="JavaScript"></li>
                 <li><img src="images\html.svg" alt="HTML"></li>
                 <li><img src="images\css.svg" alt="CSS"></li>
             </ul>
             <article class="framework">
-                <p>Frameworks</p>
+                <p role="article">Frameworks</p>
                 <hr>
-                <img src="images\sidearrow.svg">
+                <img src="images\sidearrow.svg" alt="">
             </article>
             <div class="horizontal-line"></div>
             <article class="skill">
-                <p>Skills</p>
-                <img src="images\sidearrow.svg">
+                <p role="article">Skills</p>

```

Comment:
:red_circle: please the Skills is a subheading and should be in an h tag, not a p tag. kindly fix that